assets/js/scripts.js

```diff
@@ -1297,4 +1414,59 @@ function loadFont(name, urls = {}) {
     document.fonts.add(fontFace);
     return fontFace;
   });
+}
+
+function isMobile() {
+  const isCoarsePointer = window.matchMedia("(pointer: coarse)").matches;
+  const hasTouchPoints = 'maxTouchPoints' in navigator && navigator.maxTouchPoints > 0;
+  const isMobileUAData = navigator.userAgentData ? navigator.userAgentData.mobile : false;
+  const isMobileUserAgent = /Mobi|Android|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent);
+  
+  return isCoarsePointer || hasTouchPoints || isMobileUAData || isMobileUserAgent;
+}

```

Comment:
On mozilla firefox desktop `navigator.maxTouchPoints` is equal 1 for some reason. It is probably browser bug.
Let's not disable debug setting "Override browser's search" line 1261, just leave it unchecked.